app/controllers/prototypes/newest_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Prototypes::NewestController < ApplicationController
   def index
-    @prototypes = Prototype.includes(:prototype_images).order('created_at DESC')
+    @prototypes = Prototype.includes(:prototype_images).order('created_at DESC').page(params[:page]).per(4)
```

Comment:
Line is too long. [107/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.